specification/msi/resource-manager/Microsoft.ManagedIdentity/2018-11-30/ManagedIdentity.json

```diff
@@ -388,7 +388,7 @@
         "clientSecretUrl": {
           "type": "string",
           "readOnly": true,
-          "description": " The ManagedServiceIdentity DataPlane URL that can be queried to obtain the identity credentials."
+          "description": " The ManagedServiceIdentity DataPlane URL that can be queried to obtain the identity credentials. If identity is user assigned, the clientSecretUrl is null, otherwise the clientSecretUrl is not null"
```

Comment:
> If identity is user assigned, the clientSecretUrl is null, otherwise the clientSecretUrl is not null

At the rest api level, the value of the property being `null` and the property not being present are two different things. From the examples, it seems that the response will not have the `clientSecretUrl` property, if the identity is user assigned rather then sending the property `clientSecretUrl` with `null` value.

If this is indeed the case, then can you please update the description as follows:
> If identity is user assigned, then the "clientSecretUrl" will not be present in the response, otherwise it will be present.

Reply:
@wendyNEU - Can you please take a look at my above comment?